hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/SummarySketch.java

```diff
@@ -66,8 +75,13 @@ public TableSummary add(@NonNull TableSummary left, @NonNull TableSummary right)
 
     @NonNull
     @Override
-    public Observable<PartialResult<TableSummary>> create(@NonNull ITable data) {
+    public TableSummary create(@NonNull ITable data) {
+        try {
+            Thread.sleep(1000 * Randomness.getInstance().nextInt(5));
```

Comment:
The sleep should not be here either, it was only used for testing the GUI. It will be removed.